packages/migrate/src/transforms/2571-fix-transform.ts

```diff
@@ -4,7 +4,15 @@ import { isVariableOfCatchClause, transformDiagnosedNode } from '../helpers/type
 import { isSourceCodeChanged } from '../helpers/strings';
 
 export default class FixTransform2571 extends FixTransform {
-  hint = `Object is of type '{0}'. Specify a type of variable, use type assertion: \`(variable as DesiredType)\` or type guard: \`if (variable instanceof DesiredType) { ... }\``;
+  getHint = (replacements: { [key: string]: string }) => {
+    if (Object.values(replacements).length < 1) {
+      return undefined;
+    }
+
+    const { '{0}': actualType } = replacements;
+    const hint = `Object is of type '${actualType}'. Specify a type for variable, use type assertion: \`(variable as DesiredType)\` or type guard: \`if (variable instanceof DesiredType) { ... }\``;
```

Comment:
Originally, kind of a code snippet, was provided, because real variable name was used.
The message (variable as DesiredType) is less useful than (rean_variable_name as DesiredType).